pkg/rootless/specconv/specconv_linux.go

```diff
@@ -136,3 +156,46 @@ func bindMountHostProcfs(spec *specs.Spec) error {
 
 	return nil
 }
+
+// withBindMountHostIPC replaces /dev/shm and /dev/mqueue  mount with rbind.
+// Required for --ipc=host on rootless.
+//
+// Forked from https://github.com/containerd/nerdctl/blob/v1.1.0/cmd/nerdctl/run.go#L836-L860
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
```

Comment:
☝️ I guess if we change that, we also can remove the license block (might still be worth referring to the `nerdctl` implementation for reference though.

```suggestion
// Based on https://github.com/containerd/nerdctl/blob/v1.1.0/cmd/nerdctl/run.go#L836-L860
```

Reply:
Updated